docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+you should be sure to check the display of an amp page to make sure it looks alright.
+
+When running frontend locally, the AMP version of pages can be viewed by adding the `?amp` querystring to the end of the URL. For example:
+
+```
+http://localhost:9000/world/2015/oct/15/obama-delay-withdrawal-us-troops-afghanistan?amp
+```
+
+### Amp validation on local
+Amp pages must pass validation in order to be displayed in Google search results.
+If an Amp page fails validation it is highly unlikely it will ever be served to a user.
+It is important to test Amp pages pass validation as part of any checking for frontend changes.
+
+To test this on a page run from local, request an amp page as shown above, and add the fragment `#development=1` to the end of the URL. For example:
+
+```
+http://localhost:9000/world/2015/oct/15/obama-delay-withdrawal-us-troops-afghanistan?amp#development=1
+```
+
+Before committing any frontend changes, you should also run the Amp validation tests.
+This is done by first running dev-build or article project on your local machine.
+In a new terminal tab, navigate to the frontend project and run:
+
+```
+make validate-amp
+```
+
+### Testing Amp pages on the live website
+Occasionally you might need to check something in production on the live website - for example display of a particular ad-tag
+To see an amp page, navigate to:
+                    https://amp.theguardian.com/<path-to-page>.
+
+To see the page as user from google would see it, on a mobile device (emulated or otherwise) navigate to:
+                    https://www.google.co.uk/amp/s/amp.theguardian.com/<path-to-page>
+Google will detect you are on a mobile device and serve you the amp page.
+This is useful for tests where the path to the page matters. For instance any tests for ad tags should use this method
+
+### Amp validation on the live website
+ For amp pages on the live site, you can append the "#development=1" fragment to the url to show validation in the developer console,
+   or alternatively you can enter the page url on https://ampbench.appspot.com/
+   This last is superior in the feedback it gives you as you can drill a bit more into validation errors.
+
+### Things to watch for on Amp pages
+1) That you actually are on an amp page.
+This is more relevant to testing on production where you might often be looking at the flow from
+google-search result to the amp page. The following will indicate you are on an amp page:
+    - In prod the url starts with `amp.` instead of `www.`
+    - On code and on local, the url has the `?amp` querystring described above
+    - The styling of the nav header is different
+    - The developer tools console has a message "Powered by AMP"
+2) That ads display ok.
+3) That styling, especially that of badges and graphical punctuation, such as quotes  look ok
+4) That images and other embeds display correctly - especially on narrow screens such as the iPhone5S
+
+### How to find examples of pages you want to test
```

Comment:
Should this section be further up the page and then amp as a separate section? See note above but think that would be clearer?